com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -1085,5 +1100,32 @@ private void OnDestroy()
                 Singleton = null;
             }
         }
+
+        // Command line options
```

Comment:
We already have an api for this: https://docs.unity3d.com/2023.3/Documentation/ScriptReference/DedicatedServer.Arguments.html

Reply:
I see.
@samlucas-unity could we add a conditional define that uses the DedicatedServer.Arguments API when using 2023.3?

Reply:
It's already there as UNITY_DEDICATED_SERVER_ARGUMENTS_PRESENT as part of the version defines in com.unity.netcode.runtime.asmdef

Reply:
I apologize...I wasn't clear. I see the UNITY_DEDICATED_SERVER_ARGUMENTS_PRESENT conditional define and API being used in UNET, but don't see it being used anywhere else... just didn't know if we wanted to defer to those if they are present.

Reply:
NP. Yeah I wasn't sure either, there is one for UNITY_UNET_PRESENT in there which seemed to have a similar purpose so I was following along with that. Happy to use whichever approach you think is best?

Reply:
I mean...up to you... but it would seem to simplify things if they make any changes in future versions it would/should automatically be handled.  Really, just bringing it up as something worth contemplating before merging.

Reply:
Never mind... ParseCommandOptions... I missed it.
All looks good!